README.md

```diff
@@ -24,18 +24,20 @@ CSIP depends on the [SCIP Optimization Suite](http://scip.zib.de/#scipoptsuite).
 Starting with release 0.4.0, **CSIP only supports SCIP Optimization Suite
 4.0.0** or newer.
 
-[Download](http://scip.zib.de/download.php?fname=scipoptsuite-4.0.0.tgz)
-and extract the source files, then build the shared library
-(containing SCIP and SoPlex) with
-
-    make SHARED=true GMP=false READLINE=false ZLIB=false scipoptlib
+[Download](http://scip.zib.de/download.php?fname=scipoptsuite-5.0.0.tgz)
+the SCIP Optimization Suite and extract the source files.
+Build the shared library (containing SCIP and SoPlex) with `cmake` via
+
+    mkdir build
+    cd build
+    cmake ..
```

Comment:
Does the build actually require `cmake` now? In that case, we should also update the Travis script for SCIP.jl.